.github/workflows/CI.yaml

```diff
@@ -65,6 +65,11 @@ jobs:
               conda install -y -c omnia-dev openmm --override-channels --force-reinstall
               echo "Using beta build of openeye toolkit"
               conda install -y -c openeye/label/beta openeye-toolkits --override-channels --force-reinstall;;
+            beta)
+              echo "Using beta build of OpenMM."
+              conda install -c conda-forge/label/openmm_rc -c conda-forge openmm
+              echo "Using beta build of openeye toolkit"
+              conda install -y -c openeye/label/beta openeye-toolkits --override-channels --force-reinstall;
```

Comment:
I see a syntax error in the test logs:
```
/home/runner/work/_temp/1f0bd9dd-3526-40b8-8dc7-bb695f602147.sh: line 12: syntax error near unexpected token `)'
```
Do you need to end this line with `;;` instead of just `;`?

Reply:
That was the problem!